mods/cnc/rules/defaults.yaml

```diff
@@ -491,7 +491,7 @@
 	Health:
 		HP: 400
 	Tooltip:
-		Name: Civilian Building
+		GenericName: Civilian Building
 		GenericVisibility: None
```

Comment:
We explicitly don't want the "Enemy" / "Allied" for these.  You can turn them off by setting `GenericStancePrefix: false`.